cmake/external/googletest.cmake

```diff
@@ -13,18 +13,14 @@
 # limitations under the License.
 
 include(ExternalProject)
-include(ExternalProjectFlags)
-
-ExternalProject_GitSource(
-  GOOGLETEST_GIT
-  GIT_REPOSITORY "https://github.com/google/googletest.git"
-  GIT_TAG "release-1.8.0"
-)
 
 ExternalProject_Add(
   googletest
 
-  ${GOOGLETEST_GIT}
+  DOWNLOAD_DIR ${PROJECT_BINARY_DIR}/downloads
+  DOWNLOAD_NAME googletest-1.8.0.tar.gz
+  URL https://github.com/google/googletest/archive/release-1.8.0.tar.gz
```

Comment:
(As mentioned) It's possible to use http (rather than https) here, which may help if cmake is having trouble with ssl. But unless there's a compelling reason, I'd rather just use https.

Note that the URL_HASH is (more-or-less) required even when using https. (If you omit it, cmake can't tell if the file was downloaded successfully, so it insists on re-downloading it.)

Reply:
https FTW. Let there be no unencrypted bits.